apps/wallet/src/ui/app/components/sui-apps/index.tsx

```diff
@@ -19,19 +19,16 @@ import st from './Playground.module.scss';
 
 function AppsPlayGround() {
 	const ecosystemApps = useFeature<DAppEntry[]>(FEATURES.WALLET_DAPPS).value;
-	const location = useLocation();
-
-	const queryParams = new URLSearchParams(location.search);
-	const tagFilter = queryParams.get('tagFilter');
+	const { tagName } = useParams();
 
 	const filteredEcosystemApps = useMemo(() => {
 		if (!ecosystemApps) {
 			return [];
-		} else if (tagFilter) {
-			return ecosystemApps.filter((app) => app.tags.includes(tagFilter));
+		} else if (tagName && tagName !== 'all') {
```

Comment:
Why have `/all` and why not just make the omission of this imply all?

Reply:
didn't know the path needed the `?` so `apps/` was just showing up blank, fixed